code/modules/mob/living/simple_animal/abnormality/aleph/white_night.dm

```diff
@@ -196,258 +91,386 @@ GLOBAL_LIST_EMPTY(apostles)
 			var/obj/effect/temp_visual/cult/sparks/S = new(T)
 			if(faction_check != "apostle")
 				S.color = "#AAFFAA" // Indicating that it's a good thing
-			for(var/mob/living/L in T.contents)
-				if(ishuman(L))
-					var/mob/living/carbon/human/H = L
-					new /obj/effect/temp_visual/dir_setting/cult/phase(T, H.dir)
-					addtimer(CALLBACK(src, .proc/revive_target, H, i, faction_check))
-					continue
-				// Not a human and an enemy
-				if(!(faction_check in L.faction))
-					playsound(L.loc, 'sound/machines/clockcult/ark_damage.ogg', 35, TRUE, -1)
-					L.adjustPaleLoss(holy_revival_damage)
+			for(var/mob/living/L in T)
```

Comment:
Pretty sure this allows people to get multi-hit, as I've seen multiple people report that they ARE being multi-hit by this. Which is not really good considering this can do like, 60 pale damage easily? Not as much of an issue due to the Better Regenerators change but the multi-hit pale is DEFINITELY an issue.